### DISCUSSION TITLE
How to use the value from a selection input

### DISCUSSION THREAD

**Discussion by serkonda7** [Windos/BurntToast#171] [Q&A] [answered]
I have created an input action with a selection of values. 
Now I want to get the selected value and use it further in my script but I cannot find any docs how to achieve this.
It would be great, if anyone could help me.

**Comment by Windos** [comment, marked as answer]
The unfortunate answer at the moment is that this will only be doable in PowerShell 7.1 and above.

Once I finally get time to work on BurntToast v1.0 it'll be possible in Windows PowerShell and earlier Core versions.

I've not typed it up yet, but here's some demos showing the use of input from the user:

+ [Selection Box](https://github.com/Windos/Talks/blob/main/2020/09%20-%20September/29%20-%20PSPowerHour%20-%20Croque%20Monsieur%20-%20Step%20up%20Your%20Toast%20Notification%20Game/Demos/9_SelectionBox.ps1), shows a follow up toast based on which item was selected. ([Video](https://www.youtube.com/watch?v=gkcoUDOcnGk) towards the end)
+ [Text Box](https://github.com/Windos/Talks/blob/main/2021/04%20-%20April/27%20-%20Summit%202021%20-%20Toast%20Lightning%20Demo/Demos/Action%20Toast%20-%20PS7.ps1), takes text input and surfaces it in the script. The video for this one is locked inside the PowerShell Summit 2021 event platform for now, I think it'll be public at a later date.

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
